tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "./dist/",
     "noImplicitAny": true,
-    "target": "es2015",
+    "target": "es2020",
```

Comment:
Can you say a little about why you need this?

Reply:
Just for `BigInt` literals. We can probably do without because JavaScript integers can represent 32bit integers and we locked ChocoPy values to 32 bits.